source/client/renderer/Textures.cpp

```diff
@@ -19,9 +19,19 @@ int Textures::loadTexture(const std::string& name, bool bIsRequired)
 
 	Texture t = m_pPlatform->loadTexture(name, bIsRequired);
 
-	int result = -1;
-	if (t.m_pixels)
-		result = assignTexture(name, t);
+	if (!t.m_pixels) {
```

Comment:
If the texture could not be loaded, don't generate a missing texture if the bRequired parameter is false.

Reply:
Done!